src/organizations/OrganizationEntity.js

```diff
@@ -217,7 +236,30 @@ export class OrganizationEntity extends Entity {
 		if (!this._entity || !this._entity.hasLinkByRel(rel)) {
 			return;
 		}
-
 		return this._entity.getLinkByRel(rel).href;
 	}
+
+	_getCompletionAction(enabled) {
+		const actionName = enabled ? 'track-completion' : 'do-not-track-completion';
+		return this._entity.getActionByName(actionName);
+	}
+
+	_getDisplayAction(enabled) {
+		const actionName = enabled ? 'display-progress' : 'do-not-display-progress';
+		return this._entity.getActionByName(actionName);
+	}
+
+	updateCompletionTracking(enabled) {
+		const action = this._getCompletionAction(enabled);
+		if (action) {
+			return performSirenAction(this._token, action, action.fields, false);
+		}
```

Comment:
if it returns promise in one case and undefined in another case you will not able to write `updateCompletionTracking().then()`
I think it should `return Promise.resolve(undefined);`